tool/common_mark_stats.dart

```diff
@@ -62,6 +72,60 @@ void main() {
       '– ${pct}%');
 }
 
+/// Compare two DOM trees for equality.
+bool compareHtml(List<Element> expectedEls, List<Element> actualEls) {
+  if (expectedEls.length != actualEls.length) {
+    return false;
+  }
+
+  for (int childNum in new List.generate(expectedEls.length, (i) => i)) {
```

Comment:
don't needlessly type `var childNum` etc...

Also ...just use `for (var childNum = 0; childNum < expectedElement.length; childNum++)`